python/fusion_config/query_site_search_documentation_pipeline.json

```diff
@@ -17,95 +35,17 @@
     "label" : "facet"
   }, {
     "type" : "set-params",
-    "id" : "pwp7dw5osmjz6w29",
+    "id" : "ykzta24zy5fp3nmi",
     "params" : [ {
-      "key" : "fq",
-      "value" : "_lw_data_source_s:website-lucidworks-lucidworks-docs-website OR _lw_data_source_s:website-lucidworks-lucidworks-javadocs-website",
-      "policy" : "append"
-    }, {
-      "key" : "fq",
-      "value" : "productName:fusion",
-      "policy" : "append"
-    }, {
-      "key" : "defType",
-      "value" : "edismax",
-      "policy" : "append"
-    }, {
-      "key" : "qf",
-      "value" : "title^3 body content productName productVersion",
-      "policy" : "append"
-    }, {
-      "key" : "pf",
-      "value" : "title^3 content productName productVersion",
-      "policy" : "append"
-    }, {
       "key" : "fl",
-      "value" : "id title productVersion productName og_image",
-      "policy" : "append"
-    }, {
-      "key" : "hl",
-      "value" : "true",
-      "policy" : "append"
-    }, {
-      "key" : "hl.fragsize",
-      "value" : "300",
+      "value" : "title content body subject accepted_answer_txt answer_txt",
```

Comment:
And here we're declaring hl and hl.fl params, not standard fl params or qf params.